extensions/amp-animation/amp-animation.md

```diff
@@ -27,7 +27,7 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Required Script</strong></td>
-    <td><code>&lt;script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js">&lt;/script></code></td>
```

Comment:
Are you referring to the escaped "&lt;" tags in the raw .md source? I searched through all the AMP source code and found numerous instances of escaped tags. Perhaps we need to do a wholesale fix?